python/GafferOSLTest/OSLObjectTest.py

```diff
@@ -353,17 +353,15 @@ def testCanShadeFaceVaryingInterpolatedPrimitiveVariablesAsVertex( self ) :
 		self.assertTrue( "st_add" in planeObject.keys() )
 		self.assertEqual( planeObject["st_add"].interpolation, IECore.PrimitiveVariable.Interpolation.Vertex)
 
-		# note our plane origin position (0,0,0) has a UV (0,1) because of 1.0 - t convention in cortex.
-		self.assertEqual( planeObject["st_add"].data[0], 0.0 + 1.0)
```

Comment:
I guess this is technically not an IECoreImage update, but I'll let it slide since the whole PR is basically 'adapt to Cortex 10.0.0-a2'